src/test/java/org/fcrepo/importexport/common/TransferProcessTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.importexport.common;
+
+import static java.net.URI.create;
+import static org.fcrepo.importexport.common.TransferProcess.fileForURI;
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.net.URI;
+
+import org.junit.Test;
+
+/**
+ * @author escowles
+ * @since 2016-12-14
+ */
+public class TransferProcessTest {
+
+    private File dir = new File("/export/dir/");
+    private String ext = ".ttl";
+
+    @Test
+    public void testMappingNull() throws Exception {
+        final URI uri = create("http://localhost:8080/rest/foo");
+        assertEquals(new File(dir, "rest/foo" + ext), fileForURI(uri, null, null, dir, ext));
+    }
+
+    @Test
+    public void testMappingPort() throws Exception {
+        final URI src = create("http://localhost:9090/rest/foo");
+        final URI uri = create("http://localhost:8080/rest/foo");
+        assertEquals(new File(dir, "rest/foo" + ext), fileForURI(uri, "/rest", "/rest", dir, ext));
+    }
+
+    @Test
+    public void testMappingHost() throws Exception {
+        final URI src = create("http://example.org:8080/rest/foo");
```

Comment:
`src` not used.
..therefore, this test is not testing what the name implies: `testMappingHost()`